rootfs-builder/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based in a supported based OS run:
+
+``` ./rootfs.sh <distro> ```
+
+To check the supported `rootfs` based OS use:
+
+```
+./rootfs.sh -h
+```
+
+## Adding support for new base OS ##
+
+The script `rootfs.sh` will look for directories that meets the following
+structure:
+
+- A bash script that called `rootfs_lib.sh`
+
+This file must contain a function called `build_rootfs()` this function
+must receive as first argument the path where the `rootfs` will be
+populated.
+
+
+- A bash file `config.sh`
+
+This represents the specific configuration for each based OS. It must
+provide configuration specific variables for users modify them as needed.
+The config file will be loaded before execute `build_rootfs()` to provide
+all the need configuration to the function.
+
+```bash
+#Arguments
+#rootfs_dir=$1
```

Comment:
This is great stuff, but can you format this comment block a bit to make it easier to read:

- Add a space after each `#`.
- Use "- " to represent bullets (for example in the "Expected result" section).
- Indent text that overflows a line.
- Sort the environment variables.
- Add some blank lines to space it out a little.